pkg/oc/cli/image/info/info.go

```diff
@@ -196,12 +196,22 @@ func describeImage(out io.Writer, image *Image) error {
 	case 1:
 		fmt.Fprintf(w, "Image Size:\t%s\n", units.HumanSize(float64(image.Layers[0].Size)))
 	default:
-		fmt.Fprintf(w, "Image Size:\t%s\n", fmt.Sprintf("%s in %d layers", units.HumanSize(float64(image.Config.Size)), len(image.Layers)))
+		imageSize := fmt.Sprintf("%s in %d layers", units.HumanSize(float64(image.Config.Size)), len(image.Layers))
+		if image.Config.Size == 0 {
+			imageSize = fmt.Sprintf("%d layers (size information not available)", len(image.Layers))
```

Comment:
This is too verbose.  Just print no size

Reply:
List the layers

Reply:
Thanks, updated. Layers are still being printed below:

e.g.
```
Image Size:    7 layers (no size)
Layers:        -- sha256:a02a4930cb5d36f3290eb84f4bfa30668ef2e9fe3a1fb73ec015fc58b9958b17
               -- sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4
               -- sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4
               -- sha256:5eeafb18f3a347319cc2f108f61ecf654e4c02761242a5f3f60bbd9fa4eabc44
               -- sha256:2f09d5438e83863c7d01e976ca9983507ed584644e3e22cff4866736669b7fef
               -- sha256:47ba804948df9908e95bb49dca1c771055c65cb197f9c68c34ae8f653395b512
               -- sha256:e9157828eb63c9105fad94bc56adeba441156eeb5d5ab35a352472c3c53cc6d0
```